hana/lib/HANAService.js

```diff
@@ -646,21 +646,22 @@ class HANAService extends SQLService {
     xpr({ xpr, _internal }, caseSuffix = '') {
       // Maps the compare operators to what to return when both sides are null
       const compareOperators = {
-        '=': true,
+        '==': true,
         '!=': false,
+        // REVISIT: not required for now. application must do case when statement themselves
         // These operators are not allowed in column expressions
-        '>': null,
-        '<': null,
-        '<>': null,
-        '>=': null,
-        '<=': null,
-        '!<': null,
-        '!>': null,
+        // '>': null,
+        // '<': null,
+        // '<>': null,
+        // '>=': null,
+        // '<=': null,
+        // '!<': null,
+        // '!>': null,
```

Comment:
Would that change specified behavior?

Reply:
This change only impacts the usage of these operators where `HANA` does not allow for them.

```SQL
SELECT 1>0 as gt FROM DUMMY
```

Currently considering to close this PR as https://github.com/cap-js/cds-dbs/pull/426 does the same with more fixes.